Restore.ps1

```diff
@@ -76,11 +77,33 @@ Function Get-WinNode($arch) {
         $null = mkdir $targetDir
     }
 
+    $targetDirSymbols = "$LayoutRootSymbols\tools\win-$arch"
+    if (!(Test-Path $targetDirSymbols)) {
+        $null = mkdir $targetDirSymbols
+    }
+
     Copy-Item $nodePath $targetDir
+    Copy-Item $nodePath $targetDirSymbols
+}
+
+Function Get-WinNodePdb($arch) {
+    $zipPath = Get-NetworkFile -Uri https://nodejs.org/dist/v$Version/win-$arch/node_pdb.zip -OutDir "$PSScriptRoot\obj"
```

Comment:
I note the other downloaded files include the arch and version in their filenames, making downloading them to the obj folder not overwrite anything else. But node_pdb.zip is not version or arch specific. Can we download it to a folder that is unique to that tuple so that the file doesn't overwrite another one of another architecture?
For instance, when I execute restore.ps1 twice, I'm asked to overwrite files, which I can decline and then it goes on anyway, presumably unpacking the file to the wrong locations.

Reply:
Yup, that is annoying. Should be fixed now!